Makefile

```diff
@@ -1,27 +1,38 @@
-PGRX_POSTGRES ?= pg16
-DISTNAME = $(shell grep -m 1 '^name' Cargo.toml | sed -e 's/[^"]*"\([^"]*\)",\{0,1\}/\1/')
-DISTVERSION  = $(shell grep -m 1 '^version' Cargo.toml | sed -e 's/[^"]*"\([^"]*\)",\{0,1\}/\1/')
-PG_VERSION:=16
-PGRX_PG_CONFIG =$(shell cargo pgrx info pg-config ${PGRX_POSTGRES})
+EXTENSION    = pgmq
+EXTVERSION   = $(shell grep -oP "^default_version\s*=\s\K.*" pgmq.control | tr -d "'")
+DISTVERSION  = $(EXTVERSION)
+
+DATA 		     = $(wildcard sql/*--*.sql)
+PG_CONFIG   ?= pg_config
+PGXS := $(shell $(PG_CONFIG) --pgxs)
+include $(PGXS)
+
+all: sql/$(EXTENSION)--$(EXTVERSION).sql META.json Trunk.toml
+
+sql/$(EXTENSION)--$(EXTVERSION).sql: sql/$(EXTENSION).sql
+	cp $< $@
+
+dist: Trunk.toml META.json
+	git archive --format zip --prefix=$(EXTENSION)-$(DISTVERSION)/ -o $(EXTENSION)-$(DISTVERSION).zip HEAD sql META.json Trunk.toml pgmq.control README.md UPDATING.md
 
 test:
-	cargo pgrx test $(PGRX_POSTGRES)
-	cargo test --no-default-features --features ${PGRX_POSTGRES} -- --test-threads=1 --ignored
+	cargo test --manifest-path integration_test/Cargo.toml --no-default-features -- --test-threads=1
 
-format:
-	cargo +nightly fmt --all
-	cargo +nightly clippy
+installcheck: test
 
 run.postgres:
 	docker run -d --name pgmq-pg -e POSTGRES_PASSWORD=postgres -p 5432:5432 quay.io/tembo/pgmq-pg:latest
 
-META.json: META.json.in Cargo.toml
-	@sed "s/@CARGO_VERSION@/$(DISTVERSION)/g" $< > $@
+pgxn-zip: dist
 
-$(DISTNAME)-$(DISTVERSION).zip: META.json
-	git archive --format zip --prefix $(DISTNAME)-$(DISTVERSION)/ --add-file $< -o $(DISTNAME)-$(DISTVERSION).zip HEAD
+META.json:
+	sed 's/@@VERSION@@/$(EXTVERSION)/g' META.json.in > META.json
```

Comment:
This sets `$.provides.pgmq.version`, but not `$.version`.

Reply:
It replaces both. In my META.json template  I used `@@VERSION@@` for the two fields, as for us they will always be equal. I was also thinking of removing DISTVERSION and using EXTVERSION in everything, wdyt?